Controller/Adminhtml/Config/Disconnect.php

```diff
@@ -73,15 +81,28 @@ public function __construct(
      */
     public function execute()
     {
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_APIKEY, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_EMAIL, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_CONNECTED, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_ENABLED, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_BACKUP, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_AUTH, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_SYNC, 'default', 0);
-        $this->reinitableConfig->reinit();
+        // Erase config values with the "default" scope
+        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_APIKEY);
+        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_EMAIL);
+        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_CONNECTED);
+        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_ENABLED);
+        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_BACKUP);
+        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_AUTH);
+        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_SYNC);
 
+        // Erase config values with the "websites" scope
+        $scope = \Magento\Store\Model\ScopeInterface::SCOPE_WEBSITES;
+        foreach($this->storeManager->getWebsites() as $websiteId => $website) {
+            $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_SYNC, $scope, $websiteId);
+        }
+
+        // Erase config values with the "stores" scope
+        $scope = \Magento\Store\Model\ScopeInterface::SCOPE_STORES;
+        foreach($this->storeManager->getStores() as $storeId => $store) {
+            $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_SYNC, $scope, $storeId);
```

Comment:
Good attention to detail for cleaning up the config on disconnect 👏Do we need to do this for other configuration fields such as `TAXJAR_APIKEY` and `TAXJAR_ENABLED` now that we're addressing it for multi-store instances?